las/client.py

```diff
@@ -203,14 +199,12 @@ def update_asset(
         :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
  :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
         """
-        body = {
-            'content': b64encode(content).decode() if content is not None else None,
-            'name': name,
-            'description': description,
-        }
-        return self._make_request(requests.patch, f'/assets/{asset_id}', body=dictstrip(body))
+        if content := optional_args.get('content'):
+            optional_args['content'] = b64encode(content).decode()
```

Comment:
Is there a point checking the type of content? If bytes then b64encode if str then assume it's a filepath if Path then assume it's a file path?

Reply:
Or if str assume it's already b64encoded?